src/Lykke.Service.BitfinexAdapter.Services/ShutdownManager.cs

```diff
@@ -16,12 +19,16 @@ public class ShutdownManager : IShutdownManager
     {
         private readonly ILog _log;
         private readonly List<IStopable> _items = new List<IStopable>();
-        private readonly ExchangeBase _exchange;
+        private readonly BitfinexOrderBooksHarvester _orderBooksHarvester;
+        private BitfinexAdapterSettings _settings;
+        private IComponentContext _container;
 
-        public ShutdownManager(ILog log, ExchangeBase exchange)
+        public ShutdownManager(ILog log, BitfinexOrderBooksHarvester orderBooksHarvester, BitfinexAdapterSettings settings, IComponentContext container)
```

Comment:
The second parameter could be just `IEnumerable<IStoppable>`

Reply:
Done. Refactored.